.gitignore

```diff
@@ -1,28 +1,172 @@
-node_modules
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```

Comment:
We would not store any of the VSCode config in repo. These additional lines can be removed.

Reply:
@raisedadead These four files are usually auto-generated for certain extensions (and different collaborators install different extensions) in my experience. So would you still like me to remove those 4 lines?

Reply:
> ... and different collaborators install different extensions ...

Yes, and that is the precise reason we would not store any configs on the main repo :)

Collaborators can use other strategies to store personal configs. I for instance use symlinks and version control the symlinks in my dotfiles.

You should remove these 4 lines and we are good.

Reply:
@raisedadead Yes, I understand. I added those 4 lines to prevent accidental commits and pushes of those 4 files. Was thinking in that direction. Will remove them.